packages/testing/src/BreakingChanges.spec.ts

```diff
@@ -80,14 +80,16 @@ describe('Breaking Changes', () => {
       expect(did.uri).toMatchInlineSnapshot(
         `"did:kilt:light:004quk8nu1MLvzdoT4fE6SJsLS4fFpyvuGz7sQpMF7ZAWTDoF5:z1msTRicERqs59nwMvp3yzMRBhUYGmkum7ehY7rtKQc8HzfEx4b4eyRhrc37ZShT3oG7E89x89vaG9W4hRxPS23EAFnCSeVbVRrKGJmFQvYhjgKSMmrGC7gSxgHe1a3g41uamhD49AEi13YVMkgeHpyEQJBy7N7gGyW7jTWFcwzAnws4wSazBVG1qHmVJrhmusoJoTfKTPKXkExKyur8Z341EkcRkHteY8dV3VjLXHnfhRW2yU9oM2cRm5ozgaufxrXsQBx33ygTW2wvrfzzXsYw4Bs6Vf2tC3ipBTDcKyCk6G88LYnzBosRM15W3KmDRciJ2iPjqiQkhYm77EQyaw"`
       )
+
+      expect(Did.parseDocumentFromLightDid(did.uri)).toMatchSnapshot()
```

Comment:
my idea was to use a hard-coded did here to make sure that even if we update the snapshot above bc we change the way dids are _en_coded, we still make sure we can _de_code the original form

Reply:
Sure, we can do that